### PR TITLE
Deprecate searchbird

### PR DESCRIPTION
**Pull request by simeonf**
The searchbird lesson does not work and the CLI suggestion to install a non-existent ruby lib is a potential security vulnerability to users who follow these instructions.

Remove all mention of cap utils and and highlight deprecated nature of the content.

### CODE REVIEW

File: searchbird/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Project Searchbird
 
+*Deprecated content*: this lesson no longer works due to missing dependencies.
```
Comment (agargenta) [inline searchbird/README.md]: Can you provide some additional context as to why we are missing dependencies?

Should we officially retire/archive this repo? 
Either via https://help.github.com/articles/archiving-a-github-repository/ or by moving it to https://github.com/twitter-archive/?

Comment (simeonf) [reply searchbird/README.md]: The missing dependency is a Twitter-owned gem that was unpublished when no owners stepped up to transition from ruby 1.8.7. 

We could retire this repo if you wish. I didn't want to make that call - the current status is the content lives but the tutorial in this specific section is broken. My modification notifies the end user that the content in this section is broken and removes the suggestion to the user that they install the missing and uninstallabe gem.

Comment (simeonf) [reply searchbird/README.md]: Was that "Explain to me why dependencies are missing" or amend this commit to explain to the reader why dependencies are missing?

**Comment by simeonf** [comment]
@agargenta how about the slightly longer explanation in 0315196bd1f7656bda28243561114ea57f61ffbb

**Pull request merged**